test/runtests.jl

```diff
@@ -68,92 +70,107 @@ end
     @test MOIU.supports_default_copy_to(optimizer, true)
 end
 
-const config = MOIT.TestConfig(atol=1e-3, rtol=1e-3)
-
-@testset "Basic" begin
-    @testset "Linear" begin
-        MOIT.basic_constraint_tests(
-            optimizer, config,
-            include=[
-                (MOI.SingleVariable, MOI.EqualTo{Float64}),
-                (MOI.SingleVariable, MOI.LessThan{Float64}),
-                (MOI.SingleVariable, MOI.GreaterThan{Float64}),
-                (MOI.SingleVariable, MOI.Interval{Float64}),
-                (MOI.SingleVariable, MOI.ZeroOne),
-                (MOI.SingleVariable, MOI.Integer),
-                (MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64}),
-                (MOI.ScalarAffineFunction{Float64}, MOI.LessThan{Float64}),
-                (MOI.ScalarAffineFunction{Float64}, MOI.GreaterThan{Float64}),
-                (MOI.ScalarAffineFunction{Float64}, MOI.Interval{Float64})
-        ])
-    end
-    @testset "Conic" begin
-        MOIT.basic_constraint_tests(
-            optimizer, config,
-            include=[
-                (MOI.VectorOfVariables, MOI.ExponentialCone),
-                (MOI.VectorOfVariables, MOI.DualExponentialCone),
-                (MOI.VectorOfVariables, MOI.PowerCone{Float64}),
-                (MOI.VectorOfVariables, MOI.DualPowerCone{Float64}),
-                (MOI.VectorOfVariables, MOI.SecondOrderCone),
-                (MOI.VectorOfVariables, MOI.RotatedSecondOrderCone)
-        ])
-    end
-end
-
-const bridged = MOIB.full_bridge_optimizer(optimizer, Float64)
+const config = MOIT.Config(
+    Float64, atol=1e-3, rtol=1e-3,
+    exclude=Any[MOI.ConstraintName, MOI.VariableBasisStatus, MOI.ConstraintBasisStatus], # result in errors for now
+)
 
-# Mosek errors during `MOI.set` instead of `MOI.get` when there are duplicates.
-#@testset "Name" begin
-#    MOIT.nametest(bridged)
-#end
-
-@testset "Copy" begin
-    model = MOIB.full_bridge_optimizer(Mosek.Optimizer(), Float64)
-    MOIT.copytest(bridged, model)
+@testset "Basic and linear tests" begin
+    MOIT.runtests(optimizer, config, include=["basic", "linear", "quadratic"],
+        exclude=["Indicator", "Cone", "conic"],
+    )
 end
 
-@testset "Start" begin
-    # TODO this should be checked somewhere in MOI
-    @test MOI.supports(bridged, MOI.VariablePrimalStart(), MOI.VariableIndex)
+@testset "Conic problems" begin
+    MOIT.runtests(optimizer, config,
+        include=["conic", "SecondOrderCone", "Semidefinite", "Exponential", "PowerCone", "Cone"],
+        exclude=["Indicator", "basic", "linear"],
+    )
```

Comment:
You should prefer to run these all in a single `MOI.Test.runtests` call, and explicitly exclude failing tests by their full name.

Reply:
done in last commits